cluster/gce/gci/configure-helper.sh

```diff
@@ -167,6 +169,18 @@ function main() {
   echo "Done for the configuration for kubernetes"
 }
 
+#set -x
+#mkdir -p /opt/cni/bin
+#mkdir -p /etc/cni/net.d
+#git clone https://github.com/centaurus-cloud/mizar /var/mizar
+#ln -snf /sys/fs/bpf /bpffs
+#ln -snf /var/mizar/build/bin /trn_bin
+#ln -snf /var/mizar/build/xdp /trn_xdp
+#cp -f /var/mizar/etc/cni/10-mizarcni.conf /etc/cni/net.d/10-mizarcni.conf
+#cp -f /var/mizar/build/bin/mizarcni /opt/cni/bin/mizarcni
+#ln -snf /var/mizar/build/tests/mizarcni.config /etc/mizarcni.config
+#set +x
+
```

Comment:
This section is commented. if we need this later, Please add comments.

Reply:
Removed. Using ubuntu for Mizar instead, cos will require a lot more work.